unityfiles/Assets/Scripts/Game/Enemy/EBTestMoveDown.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// Enemy Behavior test move down
+/// test behavior that moves the enemy downwards
+/// </summary>
+
+[RequireComponent (typeof (Enemy))]
+public class EBTestMoveDown : MonoBehaviour, IEnemyBehavior {
+
```

Comment:
Change the name of the class to something more readable

Reply:
EnemyBehaviorTestMoveDown er etter min mening litt langt fint om vi kann la denne gå